pkg/descheduler/evictions/evictions.go

```diff
@@ -561,6 +563,24 @@ func (pe *PodEvictor) EvictPod(ctx context.Context, pod *v1.Pod, opts EvictOptio
 	return nil
 }
 
+func (pe *PodEvictor) simulateSchedulingInDryRun(pod *v1.Pod) {
+	if !pe.dryRun {
+		return
+	}
```

Comment:
It seems to be repeated, and the outside world has already made a judgment

Reply:
Indeed. Thank you.